apps/web/components/IcelandicNamesSearcher/IcelandicNamesSearcher.tsx

```diff
@@ -255,7 +255,7 @@ export const IcelandicNamesSearcher = () => {
   const tableSection = (
     <GridColumn
       span={['12/12', '12/12', '12/12', '12/12', '3/5']}
-      paddingTop={shouldStack ? 2 : 0}
+      paddingTop={[2, 2, 2, 2, 0]}
```

Comment:
Forgot to delete the `useWindowSize` hook ... the point was to completely get rid of it :).
Thus you can delete the `shouldStack` variable as well.